models/sales/sales.yml

```diff
@@ -158,22 +160,14 @@ models:
       - name: deal_name
         description: The name you have given this deal.
 
-      - name: is_deal_pipeline_deleted
-        description: Whether the record was deleted from the deal pipeline table.
-
-      - name: is_deal_pipeline_stage_deleted
-        description: Whether the record was deleted from the deal pipeline stage table.
-
-      - name: is_deal_deleted
-        description: Whether the record was deleted from the deal table.
```

Comment:
are these fields not actually in the deal_stages model?

Reply:
These are duplicates, I noticed the same fields were already documented. So I removed these duplicate fields.

- Fields already documented [here](https://github.com/fivetran/dbt_hubspot/blob/a218221f3c9d859ef6b1606765b38d5505c4c685/models/sales/sales.yml#L98-L105).
- Duplicates that I am proposing to remove [here](https://github.com/fivetran/dbt_hubspot/blob/a218221f3c9d859ef6b1606765b38d5505c4c685/models/sales/sales.yml#L161-L168).